packages/components/src/confirm/README.md

```diff
@@ -0,0 +1,3 @@
+# `Confirm`
```

Comment:
I'm not sure about the name here. Maybe we should call it `ConfirmDialog` for now, and discuss the final name later on?

Reply:
Sounds good 👍🏻

Reply:
> I guess we need to understand better the differences between "dialog" and "modal" (I'd probably add "alert" too), and see which one applies better in this case

"Dialog" is the element type. "Modal" is a state of a dialog, which means the document should enter a new "mode" where all elements outside are inaccessible. In practice, most dialogs (including the ones generated by the system, like confirm, alert, select popups etc.) have the "modal" state. There's still no consensus on how a non-modal dialog should behave, but people usually implement them as popovers.

I'd be okay with just `Confirm`. But I'd say `ConfirmDialog` is a better name here. `window.alert` also opens a dialog, but it has a specific role of [alertdialog](https://www.w3.org/TR/wai-aria-1.2/#alertdialog). If we create such a component in the future, we should probably name it `AlertDialog`. So, to keep consistency, I'd go for `ConfirmDialog` here.

Reply:
`ConfirmDialog` it is 👍🏻